service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -48,13 +48,7 @@ public static String buildSubjectLineWithHearingBookingDateSuffix(final String s
             .collect(joining(","));
     }
 
-    private static String buildHearingDateText(final List<Element<HearingBooking>> hearingBookings) {
-        return " hearing " + dateFormatterService.formatLocalDateToString(
-            hearingBookingService.getMostUrgentHearingBooking(
-                hearingBookings).getStartDate().toLocalDate(), FormatStyle.MEDIUM);
-    }
-
-    private static String getFirstRespondentLastName(final CaseData caseData) {
+    public static String getFirstRespondentLastName(final CaseData caseData) {
```

Comment:
given this is now _public_ do we need a test scenario?

Reply:
yes